services/apps/src/apps/containers.clj

```diff
@@ -339,13 +339,13 @@
                        (where {:container_settings_id (uuidify settings-uuid)
                                :id                    (uuidify volumes-from-uuid)})))))
 
-(defn- add-volumes-from
+(defn add-volumes-from
   "Adds a record to container_volumes_from associated with the given
    container_settings UUID."
   [settings-uuid data-container-uuid]
   (insert container-volumes-from
           (values {:container_settings_id (uuidify settings-uuid)
-                   :data_containers_id     (uuidify data-container-uuid)})))
+                   :data_containers_id    (uuidify data-container-uuid)})))
```

Comment:
Except for the formatting change, the changes to this file were all made to fix some failing unit tests.